.travis.yml

```diff
@@ -3,18 +3,18 @@
   sudo: required
   language: go
   dist: trusty
-  osx_image: xcode9
+  osx_image: xcode10.1
   os:
     - linux
     - osx
   notifications:
     email: false
   go:
-    - 1.8.1
+    - 1.12.1
```

Comment:
this should be 1.12.4 now, but perhaps better to use `"1.12.x"` (with the quotes as well)

Reply:
Using 1.12.x instead of.